packages/zcli/src/commands/profiles/list.ts

```diff
@@ -25,7 +24,7 @@ export default class List extends Command {
         }
       }
     }, {
-      printLine: this.log
+      printLine: this.log.bind(this)
```

Comment:
Without `.bind(this)`, does it cause any error? It's similar to https://github.com/oclif/core/blob/main/src/cli-ux/README.md#cliuxuxtable
```
import {Command, CliUx} from '@oclif/core'
import axios from 'axios'

export default class Users extends Command {
  static flags = {
    ...CliUx.ux.table.flags()
  }

  async run() {
    const {flags} = this.parse(Users)
    const {data: users} = await axios.get('https://jsonplaceholder.typicode.com/users')

    CliUx.ux.table(users, {
      name: {
        minWidth: 7,
      },
      company: {
        get: row => row.company && row.company.name
      },
      id: {
        header: 'ID',
        extended: true
      }
    }, {
      printLine: this.log,
      ...flags, // parsed flags
    })
  }
}
```

Reply:
Yes

```
tree 7095bce4f1ecfed073df5052eb0b00d7e644f5d3
parent 66c2e8ddb06c43ab6ca3fcd3cf9bf39a490b3c58
author Frederick Zhang <frederick.zhang@zendesk.com> Wed Jul 20 11:45:21 2022 +1000
committer Frederick Zhang <frederick.zhang@zendesk.com> Wed Jul 20 16:37:56 2022 +1000

Fix profiles:list TypeError this.jsonEnabled

See [1].

[1] https://github.com/oclif/core/issues/377

diff --git a/packages/zcli/src/commands/profiles/list.ts b/packages/zcli/src/commands/profiles/list.ts
index 2facb66..2539cbf 100644
--- a/packages/zcli/src/commands/profiles/list.ts
+++ b/packages/zcli/src/commands/profiles/list.ts
@@ -23,9 +23,9 @@ export default class List extends Command {
           return log
         }
       }
     }, {
-      printLine: this.log
+      printLine: this.log.bind(this)
     })
   }
 
   async run () {
```

Reply:
ok I found https://github.com/oclif/core/issues/377 so their example is wrong 🙀

Reply:
Why read documentation when you can try a thousand things and waste 5 hours on it /s 🤪